lotsqrl/tests/utils/game.py

```diff
@@ -0,0 +1,102 @@
+import sys
+
+import lotsqrl.game
+import lotsqrl.level as level
+
+from bearlibterminal import terminal
+
+from lotsqrl.game import Options, ScreenInfo
+from lotsqrl.scenes import mainmenu
+
+
+class TestGame(lotsqrl.game.Game):
```

Comment:
This looks like copy pasta of the Game class, whats the advantage of having it?

Reply:
I needed a way to launch the game in a repeatable way as I don't want to spawn actors into walls (that would be inconvenient).
Another thing that makes it hard to use the default Game object is the fact that there is no way to disable Goblins spawning. New Goblins would influence the Behavior of your Actors. That's unwanted in a testing situation (though it would be great to be able to script that).

So basically:
 * We need to be able define the Level, instead of it being randomly generated (an empty level is good enough for now)
 * The Goblin spawning needs to be controllable (disabling it is totally fine for now)

Another use-case would be to allow for automatic Behavior performance validation. To be able to do that, we'd need a way to run the game without UI.

Other features for the future:
  * A way to run the game in a fully automatic mode